### PR TITLE
Version 0.1.0

### PR DESCRIPTION
**Pull request by andreaswittig**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by andreaswittig** [comment]
test

**Comment by andreaswittig** [comment]
test

File: README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+[![Build Status](https://travis-ci.org/cfn-modules/alerting.svg?branch=master)](https://travis-ci.org/cfn-modules/alerting)
```
Comment (michaelwittig) [inline README.md]: global replace of alerting with cloudwatch-dashboard

File: README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+## Usage
+
+```
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'cfn-modules example'
+Resources:
+  Alerting:
```
Comment (michaelwittig) [inline README.md]: Dashboard:

File: lambda-src/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+> It's important that the content of `node_modules` is part fo the repository and included in the archive published to npm!
```
Comment (michaelwittig) [inline lambda-src/README.md:1]: do we also need the node_modules files in the git repo?

**Review by michaelwittig — CHANGES_REQUESTED** [review]

**Comment by andreaswittig** [comment]
test

**Comment by andreaswittig** [comment]
test

**Comment by andreaswittig** [comment]
test

**Comment by andreaswittig** [comment]
test

**Pull request merged**